persistence/src/test/java/com/epam/esm/dao/GiftCertificateDaoImplTest.java

```diff
@@ -0,0 +1,293 @@
+package com.epam.esm.dao;
+
+import com.epam.esm.dao.exception.PersistenceException;
+import com.epam.esm.dao.extractor.GiftCertificateExtractor;
+import com.epam.esm.dao.impl.GiftCertificateDaoImpl;
+import com.epam.esm.dao.impl.TagDaoImpl;
+import com.epam.esm.dao.mapper.GiftCertificateMapper;
+import com.epam.esm.dao.mapper.TagRowMapper;
+import com.epam.esm.model.GiftCertificate;
+import com.epam.esm.model.Tag;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import javax.sql.DataSource;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
```

Comment:
Done

Reply:
Can't do that, because Intellij IDEA automatically changes multiple imports to one final when I import everything I need. (I import Set, List, HashSet, ArrayList and Collections). First screen is my 4 imports and when I add 5th final import, IDEA combines everything into one.
![1javaUtil](https://user-images.githubusercontent.com/49843180/115770738-bcfc6e00-a3b5-11eb-980c-78949fa92ee5.png)
![2javaUtil](https://user-images.githubusercontent.com/49843180/115770920-f8973800-a3b5-11eb-8e9c-9bd6d126783a.png)